<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>975</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>975</width>
    <height>600</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Yu Gothic UI</family>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <stylestrategy>PreferDefault</stylestrategy>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Snake</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>975</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>975</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::FieldsStayAtSizeHint</enum>
    </property>
    <property name="rowWrapPolicy">
     <enum>QFormLayout::DontWrapRows</enum>
    </property>
    <property name="labelAlignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>975</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>975</width>
        <height>600</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string>Game</string>
       </attribute>
       <widget class="QFrame" name="gameFrame">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>970</width>
          <height>550</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>970</width>
          <height>550</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>562</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="midLineWidth">
         <number>100</number>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>3</number>
         </property>
        </layout>
       </widget>
       <widget class="QLabel" name="scoreLabel">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>570</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Papyrus</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="timeLabel">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>570</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Papyrus</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="highScoreLabel">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>570</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Papyrus</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="timeText">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>550</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Papyrus</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Time</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="highScoreText">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>550</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Papyrus</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Best</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="scoreText">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>549</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Papyrus</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Your</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string>Menu</string>
       </attribute>
       <widget class="QPushButton" name="pushStartGame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>310</y>
          <width>200</width>
          <height>70</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>70</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#pushStartGame {
background-color: transparent;
border-image: url(:/buttons/pushStartGameNop.png);
background: none;
border: none;
background-repeat: none;
}
#pushStartGame:pressed
{
border-image:url(:/buttons/pushStartGamePressed.png);
}
#pushStartGame:hover
{
border-image: url(:/buttons/pushStartGamePressed.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushOptions">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>310</y>
          <width>200</width>
          <height>70</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>70</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#pushOptions {
background-color: transparent;
border-image: url(:/buttons/pushOptionsNop.png);
background: none;
border: none;
background-repeat: none;
}
#pushOptions:pressed
{
border-image:url(:/buttons/pushOptionsHover.png);
}
#pushOptions:hover
{
border-image:url(:/buttons/pushOptionsHover.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushExitGame">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>310</y>
          <width>200</width>
          <height>70</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>70</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#pushExitGame {
background-color: transparent;
border-image: url(:/buttons/pushExitGameNop.png);
background: none;
border: none;
background-repeat: none;
}
#pushExitGame:pressed
{
border-image:url(:/buttons/pushExitGameHover.png);
}
#pushExitGame:hover
{
border-image:url(:/buttons/pushExitGameHover.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsPause">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>250</y>
          <width>441</width>
          <height>39</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>441</width>
          <height>39</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>441</width>
          <height>39</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#graphicsPause {
background-color: transparent;
border-image: url(:/themes/themes/pause.png);
background: none;
border: none;
background-repeat: none;
}
</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsTitle">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>100</y>
          <width>669</width>
          <height>112</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>669</width>
          <height>112</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>669</width>
          <height>112</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#graphicsTitle {
background-color: transparent;
border-image: url(:/themes/themes/title.png);
background: none;
border: none;
background-repeat: none;
}
</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Result</string>
       </attribute>
       <widget class="QPushButton" name="pushRestart">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>310</y>
          <width>200</width>
          <height>70</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>70</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#pushRestart {
background-color: transparent;
border-image: url(:/buttons/pushRestartNop.png);
background: none;
border: none;
background-repeat: none;
}
#pushRestart:pressed
{
border-image: url(:/buttons/pushRestartHover.png);
}
#pushRestart:hover
{
border-image: url(:/buttons/pushRestartHover.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushExitOver">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>310</y>
          <width>200</width>
          <height>70</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>70</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#pushExitOver {
background-color: transparent;
border-image: url(:/buttons/pushExitGameNop.png);
background: none;
border: none;
background-repeat: none;
}
#pushExitOver:pressed
{
border-image:url(:/buttons/pushExitGameHover.png);
}
#pushExitOver:hover
{
border-image:url(:/buttons/pushExitGameHover.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushMenu">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>310</y>
          <width>200</width>
          <height>70</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>70</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#pushMenu {
background-color: transparent;
border-image: url(:/buttons/pushMenuNop.png);
background: none;
border: none;
background-repeat: none;
}
#pushMenu:pressed
{
border-image:url(:/buttons/pushMenuHover.png);
}
#pushMenu:hover
{
border-image:url(:/buttons/pushMenuHover.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="scoreLabelResult">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>480</y>
          <width>181</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Papyrus</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="highScoreLabelResult">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>480</y>
          <width>181</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Papyrus</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsYourScore">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>430</y>
          <width>115</width>
          <height>48</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>115</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>115</width>
          <height>48</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#graphicsYourScore {
background-color: transparent;
border-image: url(:/themes/themes/your.png);
background: none;
border: none;
background-repeat: none;
}
</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsBestScore">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>430</y>
          <width>115</width>
          <height>48</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>115</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>115</width>
          <height>48</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#graphicsBestScore {
background-color: transparent;
border-image: url(:/themes/themes/best.png);
background: none;
border: none;
background-repeat: none;
}
</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsOver">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>100</y>
          <width>530</width>
          <height>122</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>530</width>
          <height>122</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>530</width>
          <height>122</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#graphicsOver {
background-color: transparent;
border-image: url(:/themes/themes/gameOver.png);
background: none;
border: none;
background-repeat: none;
}
</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionMotyw">
   <property name="text">
    <string>Motyw</string>
   </property>
  </action>
  <action name="actionOpcje">
   <property name="text">
    <string>Opcje</string>
   </property>
  </action>
  <action name="actionNowa_gra">
   <property name="text">
    <string>Nowa gra</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
